### PR TITLE
bump(main/pandoc): 3.1.13

### PR DESCRIPTION
**Pull request by tqfx**
See https://github.com/jgm/pandoc/releases/tag/3.1.13

### CODE REVIEW

**Comment by TomJo2000** [comment]
Huh I guess nothing else depends on `pandoc`, so no reverse dependencies to bump.
(`eza` only needs it during build to make the documentation, nothing else seems to need it)
Alright, looks good then.

File: packages/pandoc/build.sh [hunk 1]
```diff
@@ -15,11 +15,11 @@ termux_step_make_install() {
 	case "$TERMUX_ARCH" in
 		aarch64)
 			srcurl="https://github.com/jgm/pandoc/releases/download/${TERMUX_PKG_VERSION}/pandoc-${TERMUX_PKG_VERSION}-linux-arm64.tar.gz"
```
Comment (TomJo2000) [inline packages/pandoc/build.sh:17]: Are we just downloading the prebuilt binaries?
How's that work, wouldn't that have conflicting hardcoded paths?

Comment (tqfx) [reply packages/pandoc/build.sh:17]:
```sh
$ pandoc -v
pandoc 3.1.13                                                     Features: +server +lua
Scripting engine: Lua 5.4
User data directory: /data/data/com.termux/files/home/.local/share/pandoc                                                           Copyright (C) 2006-2023 John MacFarlane. Web: https://pandoc.org
This is free software; see the source for copying conditions. There is no
warranty, not even for merchantability or fitness for a particular purpose.
$ file $PREFIX/bin/pandoc
/data/data/com.termux/files/usr/bin/pandoc: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, BuildID[sha1]=556d94fd1390cbc3ef7b849cce20d13c88ebb5a0, stripped
```

Comment (TomJo2000) [reply packages/pandoc/build.sh:17]: So does statically linking it magically fix it or what?
I'm still unclear how the pre-built binary is working on Termux.

Comment (tqfx) [reply packages/pandoc/build.sh:17]: The executable is statically linked and has no dynamic dependencies or dependencies on external data files.

Comment (TomJo2000) [reply packages/pandoc/build.sh:17]: Alright, thanks just wanted to confirm.

Comment (TomJo2000) [reply packages/pandoc/build.sh:17]: That's what I was worried about.

**Pull request merged**